CHANGELOG-3.5.md

```diff
@@ -148,6 +148,7 @@ Note that any `etcd_debugging_*` metrics are experimental and subject to change.
 ### gRPC Proxy
 
 - Fix [`panic on error`](https://github.com/etcd-io/etcd/pull/11694) for metrics handler.
+- Add gRPC keep alive related flags `grpc-keepalive-min-time`, `grpc-keepalive-interval` and `grpc-keepalive-timeout`.
```

Comment:
Thanks @mlmhl - couple of suggestions that please use `keepalive` as one word, and hyperlink to the PR.

Reply:
Thanks for the reminder, fixed, PTAL:)